src/main/java/org/opentripplanner/routing/algorithm/filterchain/filter/AddMinSafeTransferCostFilter.java

```diff
@@ -102,14 +97,9 @@ public String name() {
     return itineraries;
   }
 
-  @Override
-  public boolean removeItineraries() {
-    return false;
-  }
-
   @Override
   public String toString() {
-    return name() + "{minSafeTransferTimeFactor: " + minSafeTransferTimeFactor + "}";
+    return "add-min-safe-transfer-cost-filter{minSafeTransferTimeFactor: " + minSafeTransferTimeFactor + "}";
```

Comment:
Use the OTP ToStingBuilder here instead. This is no longer a "named" filter, so it o not make sence to use the old name in the toString().